packages/fxa-payments-server/src/lib/reactga-event.ts

```diff
@@ -66,18 +66,19 @@ export const ReactGALog = {
         plan_name: planName,
         product_name: productName,
       } = plan;
+      const planPrice: number = amount! / 100;
```

Comment:
Just 2 things here.
1. Can we assume that all currencies will be 2 decimal currencies? (Dividing by 100)
1. Is the non-null assertion `!` necessary here? If amount is `null` should this function handle that case?

Reply:
For posterity - 
1. Subscription Platform currently only supports currencies with decimal places. If this changes, we will need to update this.
2. Updated to remove the non-null assertion.